### PR TITLE
feat(console): organization roles table

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- init organization roles table
- add `<MultiSelect />` design system component

image

### Known issues

- Cannot edit both permissions and roles
- Search keyword not working
- Roles table styling

## Testing

tested locally

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]
stamp for the core part updates

File: packages/console/src/pages/Organizations/CreateRoleModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  useEffect(() => {
+    if (!isOpen) {
+      reset();
+    }
+  }, [isOpen, reset]);
+
+  // Initial fetch on open
+  useEffect(() => {
+    if (isOpen) {
+      void mutate();
```
Comment (charIeszhao) [inline packages/console/src/pages/Organizations/CreateRoleModal/index.tsx:74]: Loading state is going to be added later, right?

File: packages/integration-tests/src/tests/console/rbac.test.ts [hunk 2]
```diff
@@ -71,7 +71,7 @@ describe('RBAC', () => {
       text: `The API resource ${apiResourceName} has been successfully created`,
     });
 
-    await expect(page).toMatchElement('div[class$=header] div[class$=info] div[class$=name]', {
+    await expect(page).toMatchElement('div[class$=header] div[class$=info] div', {
```
Comment (charIeszhao) [inline packages/integration-tests/src/tests/console/rbac.test.ts:74]: It looks like these changes have conflict with one of my recent PRs. But nevermind you can go first

**Review by charIeszhao — APPROVED** [review]
Looks good

File: packages/console/src/ds-components/Select/MultiSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            >
+              <Close className={styles.close} />
+            </IconButton>
+          </Tag>
+        );
+      })}
+      <input
+        ref={inputRef}
+        type="text"
+        placeholder={placeholder && String(t(placeholder))}
```
[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/console/src/ds-components/Select/MultiSelect.tsx:129]: this is a raw input

[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/console/src/ds-components/Select/MultiSelect.tsx:129]: only string is allowed

File: packages/console/src/pages/Organizations/CreateRoleModal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      page: String(1),
+      page_size: String(defaultPageSize),
+      q: keyword,
+    }),
+    { revalidateOnFocus: false }
+  );
+  const [scopes] = response ?? [[], 0];
+
+  const addRole = handleSubmit(async ({ scopes, ...json }) => {
+    setIsLoading(true);
```
[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/console/src/pages/Organizations/CreateRoleModal/index.tsx:50]: yes, added to the to-do list

File: packages/console/src/pages/Organizations/CreateRoleModal/index.tsx (hunk 1 above)
Comment (gao-sun) [reply packages/console/src/pages/Organizations/CreateRoleModal/index.tsx:74]: not sure actually, will double check later

File: packages/integration-tests/src/tests/console/rbac.test.ts (hunk 2 above)
Comment (gao-sun) [reply packages/integration-tests/src/tests/console/rbac.test.ts:74]: cool ty

**Pull request merged**